cli/cli/infra/manager.py

```diff
@@ -203,12 +203,15 @@ def await_db(self, on_tick: t.Callable[[rdb.Instance], None]) -> None:
     def delete_db(self) -> None:
         """Delete the database instance."""
         # Delete the secret
-        infra.secrets.delete_db_password_secret(self.secrets)
+        infra.secrets.delete_db_password_secret_if_exists(self.secrets)
 
         # Delete the database
-        instance = self._get_database_instance_or_abort()
-        self.rdb.delete_instance(instance_id=instance.id)
-        console.print("Kong database deleted")
+        try:
+            instance = self._get_database_instance_or_abort()
+            self.rdb.delete_instance(instance_id=instance.id)
+            console.print("Kong database deleted")
+        except click.Abort:
+            logger.info("Database not found, skipping")
```

Comment:
I'd put this as a `logger.debug`, we don't want to mix logger and `console` outputs.